.github/ISSUE_TEMPLATE/issue.yaml

```diff
@@ -0,0 +1,54 @@
+name: 📖 Website & Documentation Suggestion
+description: Help us improve the Quire website and documentation
+labels: [feature]
+assignees:
+- erin-cecele
+- geealbers
+body:
+- type: markdown
+  attributes:
+    value: |
+   
+      **Thank you for opening a new issue!**
```

Comment:
Did you want to make the same heading style change here as you did in the other repo?

Reply:
I just did this but I accidentally put it in PR #430 Create pull request template for docs. 

If I merge this one first and then 430 I shouldn't have any conflicts, correct?

Reply:
Ah, I see. Yes, that should work. I think. :)

Reply:
Oops, accidentally commented as @quire-user.